packages/core/src/api/channel.ts

```diff
@@ -0,0 +1,220 @@
+/* eslint-disable jsdoc/check-param-names */
+import { makeURLSearchParams, type REST, type RawFile } from '@discordjs/rest';
+import {
+	Routes,
+	type APIChannel,
+	type APIFollowedChannel,
+	type APIInvite,
+	type APIMessage,
+	type RESTGetAPIChannelUsersThreadsArchivedResult,
+	type RESTPatchAPIChannelJSONBody,
+	type RESTPostAPIChannelInviteJSONBody,
+	type RESTGetAPIChannelMessagesQuery,
+	type RESTGetAPIChannelThreadsArchivedQuery,
+	type RESTPostAPIChannelMessageJSONBody,
+} from 'discord-api-types/v10';
+
+export class ChannelsAPI {
+	public constructor(private readonly rest: REST) {}
+
+	/**
+	 * Sends a message in a channel
+	 *
+	 * @param channelId - The id of the channel to send the message in
+	 * @param message - The options to use when sending the message
+	 */
+	public async send(channelId: string, { files, ...body }: RESTPostAPIChannelMessageJSONBody & { files?: RawFile[] }) {
```

Comment:
For consistency with the rest of the methods, I think it should be named like this
```suggestion
	public async createMessage(channelId: string, { files, ...body }: RESTPostAPIChannelMessageJSONBody & { files?: RawFile[] }) {
```